kore/src/Kore/Simplify/Data.hs

```diff
@@ -131,48 +108,6 @@ traceProfSimplify (Pattern.toTermLike -> termLike) =
             . Pretty.pretty
             <$> matchAxiomIdentifier termLike
 
-instance MonadSimplify Simplifier where
-    askMetadataTools = asks metadataTools
-    {-# INLINE askMetadataTools #-}
-
-    simplifyPattern sideCondition patt =
-        traceProfSimplify patt (Pattern.makeEvaluate sideCondition patt)
-    {-# INLINE simplifyPattern #-}
-
-    simplifyTerm = TermLike.simplify
-    {-# INLINE simplifyTerm #-}
-
-    simplifyCondition topCondition conditional = do
-        ConditionSimplifier simplify <- asks simplifierCondition
-        simplify topCondition conditional
-    {-# INLINE simplifyCondition #-}
-
-    askSimplifierAxioms = asks simplifierAxioms
-    {-# INLINE askSimplifierAxioms #-}
-
-    localSimplifierAxioms locally =
-        local $ \env@Env{simplifierAxioms} ->
-            env{simplifierAxioms = locally simplifierAxioms}
-    {-# INLINE localSimplifierAxioms #-}
-
-    askMemo = asks memo
-    {-# INLINE askMemo #-}
-
-    askInjSimplifier = asks injSimplifier
-    {-# INLINE askInjSimplifier #-}
-
-    askOverloadSimplifier = asks overloadSimplifier
-    {-# INLINE askOverloadSimplifier #-}
-
-    getCache = get
-    {-# INLINE getCache #-}
-
-    putCache = put
-    {-# INLINE putCache #-}
-
-    askSimplifierXSwitch = asks simplifierXSwitch
-    {-# INLINE askSimplifierXSwitch #-}
-
 -- | Run a simplification, returning the results along all branches.
 runSimplifierBranch ::
     Env ->
```

Comment:
IMHO this definition should always be adjacent to `runSimplifier` for readability (the two haddock comments connects the two).